### ISSUE TITLE
❌ Software Removal | Orange Website

### ISSUE THREAD

**Issue by ggg27** [privacytools/privacytools.io#1259]
# Proposed Removed Software

#### Reasons/Summary

1. Non-free/Proprietary Code

2. Over Priced[[1](https://forum.privacytools.io/t/orange-website-software-removal/1365/2)]

3. There are better alternatives
Overall list of reasons why they aren't the best or only service

#### Software's Information:

**Name**: Orange Websites
**URL**: https://www.orangewebsite.com/

**Discussion**: https://forum.privacytools.io/t/discussion-orange-website/343
**Removal Discussion**: https://forum.privacytools.io/t/orange-website-software-removal/1365b

______________

# Description
### Proprietary Code
The first and foremost reason to remove Orange Websites is that they use and recommend [non-free software](https://www.wikipedia.org/wiki/Non-free_software). Generally speaking, this is bad. However, in this instance it might have full access to the server. I personally, only use free/libre software on my VPSes. Most users do not need these services.

- https://www.orangewebsite.com/partners.php 

**Note**: I would also like to note that cPanel is not a monopoly.
https://alternativeto.net/software/cpanel/

### Cost
The second reason is cost. Orange Website starts selling VPSes at ~30 USD/month.
This is significantly above competing websites like 1984 Hosting.

### Other
There is also the concept that Orange Website might not host as high-quality products as competitors. For instance, it is unclear if they host their servers of if they are hosted by their partners [Oval Host](https://www.ovalhost.com/) and [Advania](https://advania.is/).

____________

**You forgot about _____!** Feedback is welcome.
_______________

**Related Issues**:
- https://github.com/privacytoolsIO/privacytools.io/issues/1196
- https://github.com/privacytoolsIO/privacytools.io/issues/1181

**Comment by blacklight447-ptio** [comment]
Replacing orange website with 1984 hosting would seem in place. Thoughts?

**Comment by ggg27** [comment]
@blacklight447-ptio I haven't been using 1984 for very long but they seem to run a very good ship.

______________

They seem to be large supports of free/libre software:

> 1984 commits to using Free Software wherever possible.
> `-` [About Page (3)](https://archive.is/J8Euy)

_________________

They are also used by some popular software projects like Hyperbola:

______________________
______________________

There are couple small warnings with their software.
For instance, I cannot find their logging policy:
https://1984hosting.com/tos/

Plus, their online console doesn't work in the TOR browser (WebRTC**?**)

**Comment by Mikaela** [comment]
Related to #1196, 1984 hosting doesn't mention eco friendliness while Orange does. I have no experience with either though and I would be fine with #1196 blocking hosting providers until done.

* https://www.orangewebsite.com/green-hosting.php

**Comment by Mikaela** [comment]
I propose closing this due to inactivity.

**Comment by rfb0** [comment]
> Related to #1196, 1984 hosting doesn't mention eco friendliness while Orange does. I have no experience with either though and I would be fine with #1196 blocking hosting providers until done.
> 
>     * https://www.orangewebsite.com/green-hosting.php

Related to the environmental side, it is now stated in 1984’s [about page](https://www.1984.is/about/) that they commit to use ‘only electricity from renewable energy sources and placing emphasis on reaching the highest possible energy utilization efficiency levels.’

I followed PrivacyTools recommendation and went first for Orange. I was very disappointed because they charged additional fees when paying by bank transfer and it was not possible to pay by credit card without a PayPal account. I know that either way is not privacy-friendly but I found it tricky to charge on their side a fee for bank transfers while this wasn’t stated in their Terms of Service.

I eventually canceled and tried 1984 for a shared hosting plan for my personal website. I’m quite happy about it so far. And, since they advocate for software freedom, civil rights and green energy, I’m rooting for them to be included on PrivacyTools.
